MyLibs/talk2screen.h

```diff
@@ -238,15 +242,9 @@ class HalfSplitPatch : public Patch
 
 public:
 	// Methods
-	HalfSplitPatch(
-		std::vector<float> patchRect, // bounding box
-		yDiv, // the dividing position in y
-		const char vertexPath[] = "3rdPartyLibs/OpenGL/halfSplit.vs", // path to the vertex shader file
-		const char fragmentPath[] = "3rdPartyLibs/OpenGL/halfSplit.fs" // path to the vertex fragment file
-	)
-	: shader(vertexPath, framentPath)
-	, boundBox(patchRect)
-	, yDivide(yDiv)
+	HalfSplitPatch( int yDiv ) 
```

Comment:
I don't think it would be a good idea to let users feed vs, fs filenames as arguments to the constructor, because it is not necessary and proper for users to know the implementation details.

@wanzhuoshi

Reply:
You mean the implementation details should be in the 3 inherited classes but not Patch?

Reply:
Just dont think putting the two filenames as arguments is a good idea.